designs/administrator/future-openshift/surfacing-operator-dependencies/index.md

```diff
@@ -0,0 +1,64 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Surfacing Operator dependencies (Pre- and Post-Installation) 
+There are important Operator relationships that both administrators and developers need to be aware of and understand in order to successfully search for, install, upgrade and troubleshoot inside the console. 
+
+Ultimately the goal of this work is to begin to surface dependency relationships in a way that customers can easily understand and take action on when necessary. 
+
+The designs below attempt to address things like:
+1. How can I quickly troubleshoot an operator with dependencies? Is this operator status failure also affecting other operators?
+2. What will happen if I delete an operator that another operator is dependent on?
+3. I only installed one operator, but I see that 5 additional operators are installing.
+
+## Examples of OLM dependency use cases
+- The two basic use cases:
+    1. **Direct dependency**: the Operator is dependent on other Operators in order to work. This is an actual dependency case (i.e. installing ServiceMesh also gets you Jaegar, Kiali, and Elasticsearch and all are required in order for ServiceMesh to work).  
+    2. **Package dependency**: the Operator “enables” a bulk installation of Operators so you can use those set of Operators and don't have to install them individually (but are not dependent on one another to work). The Cloud Pak case (i.e. install Cloud Pak for Integration also gets you a set of IBM Operators, but you don't necessarily need them all for one to work).
```

Comment:
My hope is that someday OLM has some mechanism to differentiate these two use cases for dependencies, because I do think they would be best served to be surfaced in different ways. In the meantime I think what's conveyed in this design would be a good start to dependency visibility and solving the most pressing issues.

Reply:
Do you think it would be worth noting these possible relationships in the UI? Like a popover next to the operator dependency graph on the details page once the operator is installed? Or do you think that will cause more confusion?